packages/node_modules/@webex/plugin-meetings/test/unit/spec/member/index.js

```diff
@@ -0,0 +1,25 @@
+import sinon from 'sinon';
+import {assert} from '@webex/test-helper-chai';
+
+import MemberUtil from '@webex/plugin-meetings/src/member/util';
+import Member from '@webex/plugin-meetings/src/member';
+
+
+describe('member', () => {
+  const sandbox = sinon.createSandbox();
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  it('test', async () => {
+    sandbox.spy(MemberUtil, 'isHandRaised');
+
+    const participant = {controls: {}};
+
+    // eslint-disable-next-line no-unused-vars
+    const member = new Member(participant);
```

Comment:
minor: you can remove the eslint comment if you remove the variable
```suggestion
    new Member(participant);
```

Reply:
Yes I tried that and then it complains about 
![image](https://user-images.githubusercontent.com/82164376/181721020-f2a88273-ea2c-4f89-8968-9ffd10c52a93.png)